### PR TITLE
Feat/Support Fish Audio

### PR DESCRIPTION
**Pull request by leng-yue**
This PR add Fish Speech (both commercial and open-sourced version) to TEN-Agent.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -32,6 +32,7 @@ COPY --from=builder /app/agents/.release/ agents/
 COPY --from=builder /app/server/bin/api /app/server/bin/api
 COPY --from=builder /usr/local/lib /usr/local/lib
 COPY --from=builder /usr/lib/python3 /usr/lib/python3
+COPY --from=builder /usr/local/bin/tman /usr/local/bin/
```
Comment (guohai) [inline Dockerfile:35]: For testing purpose, need to remove it

File: agents/ten_packages/extension/fish_audio_tts/fish_audio_tts.go [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+	if err != nil {
+		panic(err)
+	}
+	defer resp.Body.Close()
+
+	if err != nil {
+		return fmt.Errorf("TextToSpeechStream failed, err: %v", err)
+	}
+
+// Check the response status code
```
Comment (wangyoucao577) [inline agents/ten_packages/extension/fish_audio_tts/fish_audio_tts.go:102]: It seems that the code hasn't been formatted?

**Comment by wangyoucao577** [comment]
LGTM

**Review by tomasliu-agora — APPROVED** [review]
lgtm

**Pull request merged**